nilearn/reporting/_utils.py

```diff
@@ -172,67 +131,20 @@ def dataframe_to_html(df, precision, **kwargs):
     return html_table.replace('class="dataframe"', 'class="pure-table"')
 
 
-def make_stat_maps(model, contrasts, output_type="z_score"):
-    """Given a model and contrasts, return the corresponding z-maps.
-
-    Parameters
-    ----------
-    model : FirstLevelModel or SecondLevelModel object
-        Must have a fitted design matrix(ces).
-
-    contrasts : Dict[str, ndarray or str]
-        Dict of contrasts for a first or second level model.
-        Corresponds to the contrast_def for the FirstLevelModel
-        (nilearn.glm.first_level.FirstLevelModel.compute_contrast)
-        & second_level_contrast for a SecondLevelModel
-        (nilearn.glm.second_level.SecondLevelModel.compute_contrast)
-
-    output_type : :obj:`str`, default='z_score'
-        The type of statistical map to retain from the contrast.
-
-        .. versionadded:: 0.9.2
-
-    Returns
-    -------
-    statistical_maps : Dict[str, niimg]
-        Dict of statistical z-maps keyed to contrast names/titles.
-
-    See Also
-    --------
-    nilearn.glm.first_level.FirstLevelModel.compute_contrast
-    nilearn.glm.second_level.SecondLevelModel.compute_contrast
-
-    """
-    statistical_maps = {
-        contrast_id: model.compute_contrast(
-            contrast_val,
-            output_type=output_type,
-        )
-        for contrast_id, contrast_val in contrasts.items()
-    }
-    return statistical_maps
-
-
-def model_attributes_to_dataframe(model, is_volume_glm=True):
-    """Return an HTML table with pertinent model attributes & information.
+def model_attributes_to_dataframe(model):
+    """Return dataframe with pertinent model attributes & information.
 
     Parameters
     ----------
     model : Any masker or FirstLevelModel or SecondLevelModel object.
 
-    is_volume_glm : bool, optional, default=True
-        Whether the GLM model is for a volume image or not. Only relevant for
-        FirstLevelModel and SecondLevelModel objects.
-
     Returns
     -------
     attributes_df: pandas.DataFrame
         DataFrame with the pertinent attributes of the model.
     """
     if model.__class__.__name__ in ["FirstLevelModel", "SecondLevelModel"]:
-        return _glm_model_attributes_to_dataframe(
-            model, is_volume_glm=is_volume_glm
-        )
+        return _glm_model_attributes_to_dataframe(model)
```

Comment:
Could we check using `isinstance` here (at line 232) instead of string comparison?

Reply:
pretty sure I did this to avoid having a nested import to avoid circular import,
so trying to rely in something

Reply:
actually I think that if I move some function we can avoid this